### ISSUE TITLE
@category grouping

### ISSUE THREAD

**Issue by mdellanave** [pixijs/pixi-jsdoc-template#31]
Is there a way to group classes with a specific tag like @category?
I saw it working with better docs template but buy I'll prefer to use this one.

**Comment by bigtimebuddy** [comment]
I don't think this template supports that tag. But if someone wants to add support, would welcome a PR for it.

**Comment by mdellanave** [comment]
I could try following this plugin
https://github.com/ErnstHaagsman/jsdoc-plugins
but I don't know how to test it on my machine.
Can you please give me some bits?
